File: src/baseball-game/baseball-game.java
class Solution {
    public int calPoints(String[] ops) {
        List<Integer> arr = new ArrayList<>();
        for(String op : ops) {
            if(op.equals("C")) {
                arr.remove(arr.size() - 1);
            } else if(op.equals("D")) {
                arr.add(arr.get(arr.size() - 1) * 2);
            } else if (op.equals("+")) {
                arr.add(arr.get(arr.size() - 1) + arr.get(arr.size() - 2));
            } else {
                arr.add(Integer.parseInt(op));
            }
        }
        int sum = 0;
        for(int i : arr) {
            sum += i;
        }
        return sum;
    }
}

File: src/682. Baseball Game/baseball-game.js
/**
 * @param {string[]} ops
 * @return {number}
 */
var calPoints = function(ops) {
    let arr = [];
    for(let n of ops) {
        if(n == 'C') arr.pop();
        else if(n == 'D') arr.push(parseInt(arr[arr.length-1]) * 2);
        else if(n == '+') arr.push(parseInt(arr[arr.length-1])  + parseInt(arr[arr.length-2]));
        else arr.push(parseInt(n));
    }
    
    let sum = 0;
    for(let m of arr)  sum+= m; 
    return sum;
};